DQM/BeamMonitor/plugins/Vx3DHLTAnalyzer.cc

```diff
@@ -617,7 +633,11 @@ int Vx3DHLTAnalyzer::MyFit(vector<double>* vals) {
         maxTransRadius = nSigmaXY * std::sqrt(std::fabs(Gauss3D->X()[0]) + std::fabs(Gauss3D->X()[1])) / 2.;
         maxLongLength = nSigmaZ * std::sqrt(std::fabs(Gauss3D->X()[2]));
 
-        Gauss3D->Minimize();
+        try {
+          Gauss3D->Minimize();
+        } catch (cms::Exception& er) {
+          edm::LogError("Vx3DHLTAnalyzer") << "\tCaught Minuit2 exception: " << er.what();
```

Comment:
given the problems recently observed (again) with the online DQM client (see [log](https://cmsweb.cern.ch/dqm/dqm-square/tmp/content_parser_productionPARSER_job2.log)), after catching the exception and logging the error, shouldn't we return the function with an error state instead of keeping on?
I am not sure the rest of the computation make sense if the fit fails.

Reply:
@vicha-w is there a way to reproduce offline the failure reported in the data-taking [mattermost channel](https://mattermost.web.cern.ch/cms-online-ops/pl/pb99tnqwjpfsdfayqmbzpw7afr) , e.g. by making available the input streamer files that are giving rise to the issue.

Reply:
see https://github.com/cms-sw/cmssw/pull/39285

Reply:
Hi @mmusich. Sorry I didn't reach out. I have given LS files that caused the error in beamspot clients to @francescobrivio this morning.

Reply:
Hi @vicha-w sorry there is some confusion here: the streamer files you gave me this morning are **NOT** related to this issue, they are for different studies.